tests/test_utils/test_coder.py

```diff
@@ -19,3 +21,64 @@ def test_yolo_bbox_coder():
          [41.2068, -8.9232, 181.4236, 48.5840]])
     assert expected_decode_bboxes.allclose(
         coder.decode(bboxes, pred_bboxes, grid_size))
+
+
+def test_delta_bbox_coder():
```

Comment:
It is better to test boxes that are shaped as `N ×0 × 4` and `0 * 4`, I remember that the previous version is supported this shape (not very sure), and it’s best to test it and make sure this version has same behaviors.